docs/tutorials/google/start.ipynb

```diff
@@ -504,8 +505,8 @@
         "        gate_set=cirq.google.SYC_GATESET)\n",
         "\n",
         "    print(\"Scheduled. View the job at: https://console.cloud.google.com/quantum/\"\n",
-        "          \"programs{}?mods=quantum_ng2&project={}\".format(\n",
-        "              job.job_resource_name.split('/programs')[1], project_id))\n",
+        "         f\"programs/{job.program_id}/jobs/{job.job_id}\"\n",
```

Comment:
nit: shouldn't you keep the old indentation?

Reply:
Isn't this basically the same indentation?  I just changed it to a f string instead of the format.  Or do you think I should indent so the f is under the quotes?

Reply:
I think under the quotes. The number of spaces should stay fixed :-)

Reply:
Done.